GOVERNANCE.md

```diff
@@ -36,30 +41,36 @@ can directly help to shape its future.
 Potential contributors are encouraged to read the [Contributing Guide](http://zarr.readthedocs.io/en/stable/contributing.html)
 as well as the [Code of Conduct](https://github.com/zarr-developers/.github/blob/master/CODE_OF_CONDUCT.md).
 
-A community member becomes a contributor when the following criteria are met:
-
-- At least two core developers from separate employers agree membership is a good idea.
-- The new member has supported the project several times, either through code
-  or otherwise, and has demonstrated a willingness to help solve other people's
-  problems, not just their own.
-- The new member demonstrates some degree of familiarity of open source
-  practices and online courtesy.
+Anyone who has recently engaged with the project in these concrete ways, in compliance with
+the Contributing Guide and Code of Conduct, is considered a contributor.
 
 ## Core developers
 
 Core developers are community members that have demonstrated continued
-commitment to the project through ongoing contributions. They
-have shown they can be trusted to maintain Zarr with care. Becoming a
-core developer allows contributors to merge approved pull requests, cast votes
+commitment to the project through ongoing contributions to the specification,
```

Comment:
restating from above, "project" := "specification governance, and / or implementions of Zarr" above, right?